03-kubernetes&openshift/03.3-helm.md

```diff
@@ -0,0 +1,16 @@
+# Helm
```

Comment:
Might be more useful to make it about both Helm and Kustomize as they go hand in hand, and Kustomize is difficult to understand on the fly